modules/abstract-cosmos/src/lib/transaction.ts

```diff
@@ -224,13 +224,14 @@ export class CosmosTransaction extends BaseTransaction {
       case TransactionType.ContractCall:
         explanationResult.type = TransactionType.ContractCall;
         message = json.sendMessages[0].value as ExecuteContractMessage;
+        outputAmount = message.funds?.[0]?.amount ?? '0';
```

Comment:
what changed suddenly and could you add some description as well ? ticket doesn't indicate why this was needed

Reply:
Changes LGTM. But yes +1 on the above comment.